apps/native/app/src/ui/lib/checkbox/checkbox-item.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import styled from 'styled-components/native'
+import { Image } from 'react-native'
+
+import checkedIcon from '../../assets/icons/checkbox-checked.png'
+import uncheckedIcon from '../../assets/icons/checkbox-unchecked.png'
+import { dynamicColor } from '../../utils'
+import { Typography } from '../typography/typography'
+
+const Host = styled.Pressable`
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  padding-vertical: ${({ theme }) => theme.spacing[2]}px;
+  margin-horizontal: ${({ theme }) => theme.spacing[2]}px;
+  border-bottom-width: ${({ theme }) => theme.border.width.standard}px;
+  border-bottom-color: ${dynamicColor(
+    ({ theme }) => ({
+      light: theme.color.blue200,
+      dark: theme.shades.dark.shade300,
+    }),
+    true,
+  )};
+`
+
+interface CheckboxItemProps {
+  label: string
+  checked: boolean
+  onPress: () => void
+}
+
+export function CheckboxItem({ label, checked, onPress }: CheckboxItemProps) {
```

Comment:
~nit we mostly use arrow function expression in the project and island.is for consistency. Same for the datepicker

Reply:
good point, copy/paste from somewhere 😄